tests/components/zwave_js/conftest.py

```diff
@@ -650,6 +650,12 @@ def nice_ibt4zwave_state_fixture():
     return json.loads(load_fixture("zwave_js/cover_nice_ibt4zwave_state.json"))
 
 
+@pytest.fixture(name="logic_group_zdb5100_light_state", scope="session")
+def light_logic_group_zdb5100_light_state_fixture():
+    """Load the Logic Group ZDB5100 node state fixture data."""
+    return json.loads(load_fixture("zwave_js/logic_group_zdb5100_state.json"))
```

Comment:
The file includes "light" in the name. I'd rename the file, as the device is a dimmer switch and not a light, right? "logic_group_zdb5100" is enough to identify the device I think. So keep this line but just rename the json file.